test/functional/normalization/message_format_test.go

```diff
@@ -44,7 +44,7 @@ var _ = Describe("[Functional][LogForwarding][Normalization] tests for message f
 
 		// Template expected as output Log
 		var outputLogTemplate = functional.NewApplicationLogTemplate()
-		outputLogTemplate.Timestamp = nanoTime
+		outputLogTemplate.TimestampLegacy = nanoTime
```

Comment:
Why would we not want to additionally verify Timestamp = nanoTime?  Same applies in all these files